hashtable_benchmarks.cc

```diff
@@ -74,11 +74,15 @@ class alignas(kSize < 8 ? 4 : 8) Value : private Ballast<kSize - 4> {
   uint32_t value_;
 };
 
-// Use a zero cost hash function. The purpose of this benchmark is to focus on
+// Use a ~zero cost hash function. The purpose of this benchmark is to focus on
 // the implementations of the containers, not the quality or speed of their hash
-// functions.
+// functions.  Bit mixing is disabled for folly::F14*.
 struct Hash {
-  size_t operator()(size_t x) const { return x; }
+  using folly_is_avalanching = std::true_type;
+
+  size_t operator()(uint32_t x) const {
+    return (size_t{x} << 32) | x;
```

Comment:
This will run into Undefined Behavior on a 32-bit system.

Reply:
Yes. What platforms are we targeting? My change to bazel.rc will also need to be made more sophisticated to be platform-independent.

Reply:
Matt, this is a question to you. It would be nice to document the answer.

Reply:
I don't think we have tested it internally on 32 bit systems.  Since almost everything we care about is 64 bit lets start with that